examples/wave/wave-op-var-velocity.py

```diff
@@ -126,6 +129,16 @@ def rk4_step(y, t, h, f):
     return y + h/6*(k1 + 2*k2 + 2*k3 + k4)
 
 
+def estimate_rk4_timestep(dcoll, c, scaling=None):
```

Comment:
Not a huge fan of this scaling parameter. Users might as well multiply the output by something.

Reply:
Got rid of these in https://github.com/inducer/grudge/pull/108/commits/b2a8a35c8366c156d30b9a409373b9b505398d16